brewchat/build.gradle

```diff
@@ -33,6 +33,11 @@ android {
         apiKey System.getenv("TESTFAIRY_KEY")
     }
 
+    packagingOptions {
+        exclude 'META-INF/LICENSE.txt'
+        exclude 'META-INF/NOTICE.txt'
+    }
+
```

Comment:
There were some conflicting files between two of the JARs we were using. No big deal.